docs/source/configuration/multinode_cassandra.rst

```diff
@@ -0,0 +1,112 @@
+=================================
+Multinode Cassandra HA Guarantees
+=================================
+
+A multinode Cassandra cluster can generally be in one of three states:
+
+    1. All nodes are up.
+    #. Nodes down, but there is a quorum of up nodes (more than half the nodes are up).
```

Comment:
It's a little more complex than that, if you allow cases where the RF is less than the number of nodes.

For example, if you have a 7 node cluster with RF3, you can still only afford to have one node down to guarantee quorum (although quorum writes _may_ succeed with two nodes down, provided your data is not on both nodes). Bumping the RF to 5 means you can have two nodes down, but only with RF 7 can you have three nodes down.